applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.hpp

```diff
@@ -236,6 +236,7 @@ namespace Kratos
 
 	private:
 		Quaternion<double> morientation;

```

Comment:
if you have the quaternion why do you need the MemberRotationMatrix?

Reply:
I dont know why but if I eg give a matrix([[0,1,0],[1,0,0],[0,0,1]]) and transform it to a quaternion and then call it later to give me the matrix it returns not zeros but small numbers as 10^⁻16. If I have large rotations this inaccuracy leads to instabilities and divergence in some cases. But you are right I forgot to delete the Quaternion. Actually I can get rid of the whole orientation class now as I avoid the Quaternion